src/patternfly/components/Hint/hint.scss

```diff
@@ -0,0 +1,114 @@
+.pf-c-hint {
+  --pf-c-hint__header--GridColumnWidth: auto;
+  --pf-c-hint__actions--GridColumnWidth: calc(var(--pf-global--spacer--2xl) + #{pf-size-prem(44px)});
+  --pf-c-hint--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-hint--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-hint--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-c-hint--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-hint--BackgroundColor: var(--pf-global--palette--blue-50);
+  --pf-c-hint--BorderColor: var(--pf-global--palette--blue-100);
+  --pf-c-hint--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-hint--BoxShadow: var(--pf-global--BoxShadow--sm);
+  --pf-c-hint--Color: var(--pf-global--Color--100);
+
+  // Hint Header
+  --pf-c-hint__header--FontSize: var(--pf-global--FontSize--lg);
+
+  // Hint Body
+  --pf-c-hint__body--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-hint__header--body--MarginTop: var(--pf-global--spacer--md);
+
+  // Hint Footer
+  --pf-c-hint__footer--child--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-hint__header--footer--MarginTop: var(--pf-global--spacer--md);
+  --pf-c-hint__body--footer--MarginTop: var(--pf-global--spacer--md);
+
+  // Hint Actions
+  --pf-c-hint__actions--MarginLeft: var(--pf-global--spacer--2xl);
+  --pf-c-hint__actions--c-dropdown--MarginTop: calc(var(--pf-global--spacer--form-element) * -1);
+
+  // Dominant modifier
+  --pf-c-hint--m-dominant--BackgroundColor: var(--pf-global--primary-color--100);
+  --pf-c-hint--m-dominant--BoxShadow: var(--pf-global--BoxShadow--md);
+  --pf-c-hint--m-dominant--Color: var(--pf-global--Color--light-100);
+
+  display: grid;
+  grid-template-columns: var(--pf-c-hint__header--GridColumnWidth) var(--pf-c-hint__actions--GridColumnWidth);
+  padding: var(--pf-c-hint--PaddingTop) var(--pf-c-hint--PaddingRight) var(--pf-c-hint--PaddingBottom) var(--pf-c-hint--PaddingLeft);
+  color: var(--pf-c-hint--Color);
+  background-color: var(--pf-c-hint--BackgroundColor);
+  border: var(--pf-c-hint--BorderWidth) solid var(--pf-c-hint--BorderColor);
+  box-shadow: var(--pf-c-hint--BoxShadow);
+
+  &.pf-m-dominant {
+    border: unset;
```

Comment:
If you're just disabling the border, `border: 0;` is probably better. `unset` is like a 2 step evaluation depending on whether the property is inheritable. It will evaluate to `initial` for `border`.

You could also do `--pf-c-hint--BorderWidth: 0;`